tests/models/neural_networks/test_linear_network.py

```diff
@@ -44,18 +50,41 @@ def mocktrain(self):
         assert model_dict['include_pred_month'] == include_pred_month
         assert model_dict['surrounding_pixels'] == surrounding_pixels
 
-    @pytest.mark.parametrize('use_pred_months', [True, False])
-    def test_train(self, tmp_path, capsys, use_pred_months):
+    @pytest.mark.parametrize(
+        'use_pred_months,experiment',
+        [(True, 'one_month_forecast'),
+         (False, 'one_month_forecast'),
+         (False, 'nowcast'),
+         (True, 'nowcast')]
+    )
+    def test_train(self, tmp_path, capsys, use_pred_months, experiment):
+        # make the x, y data (5*5 latlons, 36 timesteps, 1 feature)
         x, _, _ = _make_dataset(size=(5, 5), const=True)
         y = x.isel(time=[-1])
 
-        test_features = tmp_path / 'features/train/hello'
-        test_features.mkdir(parents=True)
-
-        norm_dict = {'VHI': {'mean': np.zeros(x.to_array().values.shape[:2]),
-                             'std': np.ones(x.to_array().values.shape[:2])}
-                     }
-        with (tmp_path / 'features/normalizing_dict.pkl').open('wb') as f:
+        test_features = tmp_path / f'features/{experiment}/train/hello'
+        test_features.mkdir(parents=True, exist_ok=True)
+
+        # if nowcast we need another x feature
+        if experiment == 'nowcast':
+            x_add1, _, _ = _make_dataset(size=(5, 5), const=True, variable_name='precip')
```

Comment:
There's no reason multi-variable testing should be restricted to the nowcasting case

Reply:
good point